docs/Server.md

```diff
@@ -154,6 +154,21 @@ The header name used to know the request id. See [the request id](https://github
 
 + Default: `'request-id'`
 
+<a name="factory-gen-request-id"></a>
+### `genReqId`
+
+Function for generating the request id. It will receive the incoming request as a parameter.
```

Comment:
`Function for generating the request id`=> `Function for generating the request id if the request id header is not in the request`?

Reply:
Sorry, my bad. This is not true. If the request id generator is provided, the id is not read from request header. Could you undo this change?

Reply:
actually it is `req.headers[requestIdHeader] || (nextReqId = (nextReqId + 1) & maxInt)` in the default generator function implementation so I guess it is true?

Reply:
But if you override it, then yes the id is not read from request header unless you implement it that way

Reply:
Undoing the change and adding `+ Default: `value of 'request-id' if provided or monotonically increasing integers`